packages/blade/src/components/Accordion/StyledAccordionButton.web.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from './commonStyles';
 import { castWebType } from '~utils';
 
-const StyledAccordionButton = styled.div<StyledAccordionButtonProps>((props) => {
+const StyledAccordionButton = styled.button<StyledAccordionButtonProps>((props) => {
```

Comment:
We can't use BaseButton? Is this because you want the markdown to be button but don't want the styles?

Reply:
Yup exactly, the styles are quite different from BaseButton's